src/js/angular/core/directives/queryeditor/query-editor.directive.js

```diff
@@ -875,6 +866,7 @@ function queryEditorDirective($timeout, $location, toastr, $repositories, Sparql
                     scope.currentQuery.sameAs = toBoolean(sameAs);
                 }
                 autoexecuteQueryIfRequested();
+                $location.search({});
```

Comment:
If you do this, then the previous url location would remain in the browser history which will be bad for the user experience.

Reply:
fixed